java-lib/src/main/java/com/wavefront/metrics/JsonMetricsReporter.java

```diff
@@ -148,14 +148,13 @@ public void reportMetrics() {
         builder.port(sunnylabsPort);
       }
       builder.queryParam("h", host);
-      builder.queryParam("t", table);
```

Comment:
is removing this enough or do we need to specify the tenant=collector here?

Reply:
Removing should be fine, we don't need to specify the tenant.